### PR TITLE
Add support for new http payload annotation fix

### PR DESCRIPTION
**Pull request by aneeshafedo**
## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? yes
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

### CODE REVIEW

File: openapi-cli/src/main/java/io/ballerina/openapi/cmd/OASContractGenerator.java [hunk 1]
```diff
@@ -81,8 +83,13 @@ public List<OpenAPIConverterDiagnostic> getErrors() {
     public void generateOAS3DefinitionsAllService(Path servicePath, Path outPath, String serviceName,
                                                   Boolean needJson) {
         // Load project instance for single ballerina file
-        project = ProjectLoader.loadProject(servicePath);
-        Package packageName = project.currentPackage();
+        String customBallerinaHome = System.getProperty("ballerina.home");
+        Path ballerinaHome = customBallerinaHome == null ? null : Path.of(customBallerinaHome);
+        Environment environment = EnvironmentBuilder.getBuilder().setBallerinaHome(ballerinaHome).build();
```
Comment (NipunaRanasinghe) [inline openapi-cli/src/main/java/io/ballerina/openapi/cmd/OASContractGenerator.java:88]: I suppose we don't have to manually set the ballerina home. shall we use the default environment?

File: openapi-cli/src/test/java/io/ballerina/openapi/generators/openapi/ParameterAnnotationTests.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
+package io.ballerina.openapi.generators.openapi;
```
Comment (NipunaRanasinghe) [inline openapi-cli/src/test/java/io/ballerina/openapi/generators/openapi/ParameterAnnotationTests.java:1]: Missing license header

Comment (aneeshafedo) [reply openapi-cli/src/test/java/io/ballerina/openapi/generators/openapi/ParameterAnnotationTests.java:19]: Added.

**Review by NipunaRanasinghe — COMMENTED** [review]
Shall we keep a separate issue issue to track the support for the union types related scenarios? or do we already one?

**Review by NipunaRanasinghe — APPROVED** [review]
LGTM

**Review by lnash94 — APPROVED** [review]

**Pull request merged**